lti_auth/auth.py

```diff
@@ -4,6 +4,7 @@
 from pylti.common import LTIException
 
 from lti_auth.lti import LTI
+from nameparser import HumanName
```

Comment:
nameparser looks handy. probably some other projects that could be useful on too.